doc/generic/pgf/text-en/pgfmanual-en-base-shadings.tex

```diff
@@ -47,12 +47,86 @@ \subsection{Overview}
 This line means that at 0cm (the left end) of the bar, the color should be red,
 which has red-green-blue (rgb) components (1,0,0). At 2cm, the bar should be
 green, and at 4cm it should be blue. Instead of |rgb|, you can currently also
-specify |gray| as color model, in which case only one value is needed, or
+specify |cmyk| as color model, in which case four values are needed,
+|gray| as color model, in which case only one value is needed, or
 |color|, in which case you must provide the name of a color in parentheses. In
 a color specification the individual specifications must be separated using a
 semicolon, which may be followed by a whitespace (like a space or a newline).
 Individual specifications must be given in increasing order.
 
+\subsubsection{Color models}
+
```

Comment:
I hope I can review this tomorrow.  In the meantime, please add your copyright to all the files and also a short note
```
\subsubsection{Color models}

\noindent\emph{by David Purton}}
```
in the section in the manual (if you want).  I think you deserve credit for this.

Reply:
I'll add something like:

```
% CMYK and grayscale shadings adaptation copyright 2019 by David Purton
```

to all the files (except `pgfsys.code.tex`). Is that the kind of thing you have in mind? Though the bulk of the changes are really only found in `pgfcoreshade.code.tex`.